litmus-portal/frontend/src/views/Analytics/Overview/index.tsx

```diff
@@ -242,7 +242,7 @@ const Overview: React.FC = () => {
           src="./icons/dashboardCloud.svg"
           alt="Schedule a workflow"
           heading="Configure a chaos interleaved dashboard"
-          description="Data source(s) have been found to be connected in this project. Select “Add dashboard” to configure a chaos interleaved dashboard"
+          description="Data source(s) have been found to be connected in this project. Select “Create dashboard” to configure a chaos interleaved dashboard"
```

Comment:
May this be taken from translation

Reply:
As discussed with @vanshBhatia-A4k9 translation for overview and workflow dashboards were to be added in a single PR many files in both the sections are currently missing those. @whitetiger1399 Thats not the core intent of this PR

Reply:
cc: @Saranya-jena @SarthakJain26